main.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 var (
-	conf = flag.String("conf", "config.yaml", "The config path file")
-	addr = flag.String("metrics-address", ":2112", "The address to listen on for HTTP requests.")
+	conf    = flag.String("conf", "config.yaml", "The config path file")
+	addr    = flag.String("metrics-address", ":2112", "The address to listen on for HTTP requests.")
+	tlsConf = flag.String("metrics-tls-config", "", "The TLS config file for your metrics.")
```

Comment:
Nice addition 👍 would it be better to have a config section for metrics instead? (have to duplicate `addr`  for backward compatibility atm) so the arg key section is not growing.
```
metrics:
  - servePort: 2112
  - namePrefix: bla
  - tlsConfigFile: /path
```